### PR TITLE
JetPack 5.1.1/L4T R35.3.1 update

### PR DESCRIPTION
**Pull request by madisongh**
Still a work in progress, with much testing to do.

So far, I've successfully built, booted, and run a handful of basic tests on `demo-image-full` images for
* jetson-agx-xavier-devkit
* jetson-orin-nano-devkit  (with rootfs on SDcard)

### CODE REVIEW

**Comment by ichergui** [comment]
@madisongh
I tried the `Jetson AGX Orin` and it is working fine
I created a new spreadsheet regarding the new [Jetpack-5.1.1\ L4T-R35.3.1](https://docs.google.com/spreadsheets/d/1EWG5QZvtWdk74gz48D4xCbegK-bsqMFJ8XOsyGdUUeI/edit#gid=1125662453)
I will continue with the testing and will keep you posted

File: recipes-bsp/tegra-binaries/tegra-helper-scripts/tegra234-flash-helper.sh [hunk 1]
```diff
@@ -366,7 +371,10 @@ if [ "$BOARDID" = "3701" ]; then
     case $chip_sku in
 	00)
         ;;
-	90|97|9E)
+	90)
+            BPF_FILE=$(echo "$BPF_FILE" | sed -e"s,T.*-A1,TA992M-A1,")
```
Comment (ibstewart) [inline recipes-bsp/tegra-binaries/tegra-helper-scripts/tegra234-flash-helper.sh:375]: Typo, should be `TE992M-A1`

Comment (ichergui) [reply recipes-bsp/tegra-binaries/tegra-helper-scripts/tegra234-flash-helper.sh:375]: Thanks @ibstewart

**Review by ichergui — APPROVED** [review]
LGTM. Thanks @madisongh

**Pull request merged**